WPMedia/fusion-news-theme-blocks
# PEN-1657 Top Table List spacing broken

**If you have not filled out the checklist below, the pr is not ready for review.**

## Description
We are seeing in Liberation that the articles in the top table list are not spaced properly around the dividers. Some images have no space to the divider at all, some very little. Space from vertical items also looks too small.

Libe example top left: http://liberation-liberation-sandbox.cdn.arcpublishing.com/

I found the same behavior on core components: http://liberation-liberation-sandbox.cdn.arcpublishing.com/

## Jira Ticket
- [PEN-1657](https://arcpublishing.atlassian.net/browse/PEN-1657)

## Acceptance Criteria
Spacing should match design specifications: https://app.zeplin.io/project/5ea04773df4e1b7ae81167f8/dashboard?sid=5eb17359fd3f6c19c93f505a - Can't find link 
Expected Behavior
Spacing should be evenly and not too small (like before)

## Test Steps
1. In Fusion-News-Theme repo, checkout master & git pull
2. After checking out this feature branch in fusion-news-theme-blocks, run rm -rf ./node_modules && npx lerna clean && npm i && npx lerna bootstrap && cd blocks/top-table-list-block && npm i && cd ../..
3. Run: npx fusion start -f -l @wpmedia/top-table-list-block
4. Open any Top Table List and test expected result (example: https://corecomponents.arcpublishing.com/pagebuilder/editor/curate?p=pLtpiR1biLVrZr&v=vAN3yH1biLVrZr)

## Effect Of Changes
### Before
![image](https://user-images.githubusercontent.com/47773457/105454745-fb01cd00-5cb4-11eb-942a-16b4ce0b3dd3.png)

### After
Desktop version:
![image](https://user-images.githubusercontent.com/47773457/105454787-09e87f80-5cb5-11eb-9ce9-21b1aa44b2cf.png)

Based on Zeplin:
![image](https://user-images.githubusercontent.com/47773457/105454823-1e2c7c80-5cb5-11eb-94e0-6a1b1d02e942.png)

Mobile version:
![image](https://user-images.githubusercontent.com/47773457/105683062-cb0d3080-5f25-11eb-82f9-e7a9295d6b80.png)
Zeplin:
![image](https://user-images.githubusercontent.com/47773457/105683133-deb89700-5f25-11eb-9a8b-2d26da87a28c.png)

## Dependencies or Side Effects
_Examples of dependencies or side effects are:_
- Additional settings that will be required in the blocks.json
- Changes to the custom fields which will require users to reconfigure features
- Update to css framework or SDK
- Dependency on another PR that needs to be merged first

## Author Checklist
_The author of the PR should fill out the following sections to ensure this PR is ready for review._
please explain why (so that we can fix it whenever it gets refactored).

## Reviewer Checklist 
_The reviewer of the PR should copy-paste this template into the review comments on review._
